rapid7/metasploit-framework
# CVE-2020-5741: Plex rce on Windows

This PR adds an exploit against Plex for Windows.  An authenticated user can create a new photo library, and upload a python pickled `Dict` file.  After setting an advanced hidden variable and rebooting the plex service, the `Dict` file is unpickled (and executed).  We use this to then call our payload file.  RCE.

## Verification
  1. Install the application
  2. Register/Connect it to your Plex Pass account
  3. Start msfconsole
  4. Do: ```use windows/http/plex_unpickle_dict_rce```
  5. Do: ```run```
  6. You should get a shell.
